crates/sui-rpc-api/proto/sui.node.v2.proto

```diff
@@ -0,0 +1,352 @@
+syntax = "proto3";
+
+package sui.node.v2;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/timestamp.proto";
+import "sui.types.proto";
+
+//
+// Rpc Node interface
+//
+
+service Node {
+  rpc GetNodeInfo(google.protobuf.Empty) returns (GetNodeInfoResponse);
+  rpc GetCommittee(GetCommitteeRequest) returns (GetCommitteeResponse);
+
+  rpc GetObject(GetObjectRequest) returns (GetObjectResponse);
+  rpc GetTransaction(GetTransactionRequest) returns (GetTransactionResponse);
+  rpc GetCheckpoint(GetCheckpointRequest) returns (GetCheckpointResponse);
+  rpc GetFullCheckpoint(GetFullCheckpointRequest) returns (GetFullCheckpointResponse);
+
+  rpc ExecuteTransaction(ExecuteTransactionRequest) returns (ExecuteTransactionResponse);
+}
+
+message GetNodeInfoResponse {
+  // The chain identifier of the chain that this Node is on
+  optional sui.types.Digest chain_id = 1;
```

Comment:
It looks like in proto3 all fields are optional by default (https://stackoverflow.com/questions/42622015/how-to-define-an-optional-field-in-protobuf-3), do we still need to mark them explicitly as such?